public/resources/js/directives/api-list.js

```diff
@@ -76,7 +76,7 @@ angularIO.directive('apiList', function () {
         { cssClass: 'function', title: 'Function', matches: ['function'] },
         { cssClass: 'enum', title: 'Enum', matches: ['enum'] },
         { cssClass: 'type-alias', title: 'Type Alias', matches: ['type-alias'] },
-        { cssClass: 'const', title: 'Const', matches: ['var', 'let', 'const'] }
+        { cssClass: 'const', title: 'Const', matches: ['const', 'var', 'let'] }
```

Comment:
This change is necessary so that `'const'` elements get picked up in line 215; because `'const'` is used as the canonical representative for the matches array elements.